runners/portability/java/build.gradle

```diff
@@ -214,7 +214,7 @@ def createUlrValidatesRunnerTask = { name, environmentType ->
   }
 
   if (environmentType == "DOCKER") {
-    vrTask.dependsOn ":sdks:java:container:docker"
+    vrTask.dependsOn ":sdks:java:container:java8:docker"
```

Comment:
Are there plans to make this and the other portable tests runnable with the Java 11 container? Seems like we should drop a TODO and/or file a jira